openmetadata-ui/src/main/resources/ui/public/locales/en-US/Messaging/CustomMessaging.md

```diff
@@ -1,23 +1,36 @@
-# CustomMessaging
+# Custom Messaging
 
-In this section, we provide guides and references to use the CustomMessaging connector.
+In this section, we provide guides and references to use the Custom Messaging connector.
 
-# Requirements
-<!-- to be updated -->
-You can find further information on the Kafka connector in the [docs](https://docs.open-metadata.org/connectors/messaging/custommessaging).
+Note that this connector is a wrapper for any Python class you create and add to the OpenMetadata ingestion image. The
+full idea around it is bringing you the tools to bring into OpenMetadata any source that is only available within
+your business/engineering context.
+
+You can learn more about Custom Connectors and see them in action in the following [Webinar](https://www.youtube.com/watch?v=fDUj30Ub9VE&ab_channel=OpenMetadata).
+Also, you can directly jump to the demo code [here](https://github.com/open-metadata/openmetadata-demo/tree/main/custom-connector).
 
 ## Connection Details
 
 $$section
 ### Source Python Class $(id="sourcePythonClass")
 
-Source Python Class Name to instantiated by the ingestion workflow
-<!-- sourcePythonClass to be updated -->
-$$
+Source Python Class Name to instantiated by the ingestion workflow.
+
+Note that it should implement the `next_record` method so that the Workflow can keep reading and sending records
+to the OpenMetadata API.
 
 $$section
 ### Connection Options $(id="connectionOptions")
 
-Additional connection options to build the URL that can be sent to service during the connection.
-<!-- connectionOptions to be updated -->
+This property becomes useful when we need to send input parameters to our Source Class.
+
+If, for example, we want to run a piece of logic based on the value of a parameter named `business_unit`, we can
+pass the key `business_unit` with any value, and read it in the Source via:
```

Comment:
The manual break lines in these markdown files work differently. I would double-check if they look correct when changing the component's width where they are contained.

Reply:
The same comments for other places where they have been used.

Reply:
IMO let's see if we can clean this on the UI itself to adhere to typical md syntax. Otherwise, tomorrow I'll do a review of all the docs we have here to align them